community/tools/graphql2chartjs/example/app/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
This is a stock create-react-app readme. Shouldn't we add our app's instructions here?

Reply:
Like migration apply etc.?